### PR TITLE
Bug 1949866: use initialization structure for ID provider

### PR DESCRIPTION
**Pull request by joelddiaz**
fix param init for create-identity-provider
fix param init for create-iam-roles

### CODE REVIEW

File: pkg/cmd/provisioning/aws/create_identity_provider.go [hunk 1]
```diff
@@ -11,14 +11,15 @@ import (
...
-	"github.com/openshift/cloud-credential-operator/pkg/cmd/provisioning"
 	"io/ioutil"
 	"log"
 	"net/url"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/openshift/cloud-credential-operator/pkg/cmd/provisioning"
+
```
Comment (akhil-rane) [inline pkg/cmd/provisioning/aws/create_identity_provider.go:22]: this import should come last, along with other openshift import

Comment (joelddiaz) [reply pkg/cmd/provisioning/aws/create_identity_provider.go:22]: ahh...VS Code...i almost with it wouldn't auto-import things

**Pull request merged**